src/main/java/org/yomigae/model/TempleModel.java

```diff
@@ -0,0 +1,171 @@
+package org.yomigae.model;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.Collections;
+import java.util.Arrays;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Iterables;
+
+import heronarts.lx.model.LXPoint;
+import heronarts.lx.model.LXModel;
+
+/**
+ * Class to encapsulate temple structure and points.
+ *
+ * Glossary:
+ *  - torii: gate structure comprised of two columns and a beam.
+ *  - torii types: T1 through T6, indicate the relative height of the torii
+ *  - column: vertical components of a torii
+ *  - column opening-width: head-on inside length between columns of a torii
+ *  - column width: head-on width of a torii's column
```

Comment:
Just to be clear, "head-on" in this case means "looking through the tunnel and hall (along the 12/6 o'clock axis)", while "side-view" means  "looking along the 3/9 o'clock axis", correct?

And on the XYZ orientation: it looks like you're using X as the 12/6 axis and Z as the 3/9?

Reply:
Yes I had it rotated incorrectly in my head when I first started writing this class. The comment should be updated, I'll do this when I also change the other variable names probably.

I believe what I had in mind by "head-on" was if you are looking at a torii as if you are to walk into the gate.

X is 6->12, Y is up, and Z is 3->9; this should also be documented.